SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceUniqueId.h

```diff
@@ -12,19 +12,21 @@ class UFunction;
 
 namespace SpatialGDK
 {
-struct EventTraceUniqueId
+class EventTraceUniqueId
```

Comment:
Had a bug where I passed an integer into EventTraceUniqueId directly which was not intentional behavior